modules/docs/docgen/createDocProgram.js

```diff
@@ -34,8 +34,10 @@ function getTSConfig(basePath = '.') {
     const contents = ts.readConfigFile(tsconfigPath, ts.sys.readFile);
     if (contents.config) {
       // https://github.com/microsoft/TypeScript/issues/5276#issuecomment-149369652
-      config = ts.convertCompilerOptionsFromJson(contents.config.compilerOptions, tsconfigPath)
-        .options;
+      config = ts.convertCompilerOptionsFromJson(
+        contents.config.compilerOptions,
+        tsconfigPath
+      ).options;
```

Comment:
This is a prettier change since we upgraded to Prettier v2. First change to this file since then.